### PR TITLE
docs(v2): Reorganize migration guide

### PR DESCRIPTION
**Pull request by wgao19**
## Motivation

- reorganize migration guide v1 -> v2
- address #1549, add guides around `<AUTOGENERATED_TABLE_OF_CONTENTS>`

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

NA

## Related PRs

NA

### CODE REVIEW

**Comment by endiliey** [comment]
Thank you :) let me know when its no longer WIP

**Comment by wgao19** [comment]
@endiliey I'm not very familiar with remark-toc, do you mind adding the guide here?

**Comment by endiliey** [comment]
I think just writing 

it has been deprecated, you might want to use [remark-toc](https://github.com/remarkjs/remark-toc) instead and pass it to docs plugin `remarkPlugins` option

**Comment by wgao19** [comment]
what really, dont we want to include how to pass that...

**Comment by endiliey** [comment]
Actually using remark-toc isnt really a direct workaround. Its a different syntax. Just point that it has been deprecated, and link to that github issue instead for reasoning.

File: website/docs/migrating-from-v1-to-v2.md [hunk 1]
```diff
@@ -408,7 +449,11 @@ Please refer to [creating pages](creating-pages.md) to learn how Docusaurus 2 pa
   - Index page - [Flux](https://github.com/facebook/flux/blob/master/website/src/pages/index.js) (recommended), [Docusaurus 2](https://github.com/facebook/docusaurus/blob/master/website/src/pages/index.js), [Hermes](https://github.com/facebook/hermes/blob/master/website/src/pages/index.js),
   - Help/Support page - [Docusaurus 2](https://github.com/facebook/docusaurus/blob/master/website/src/pages/help.js), [Flux](http://facebook.github.io/flux/support)
 
-## Update your docs
+## Content
+
+### Remove `<AUTOGENERATED_TABLE_OF_CONTENTS>`
```
Comment (endiliey) [inline website/docs/migrating-from-v1-to-v2.md]:
```suggestion
### Remove AUTOGENERATED_TABLE_OF_CONTENTS
```

MDX has weird stuff dealing with unclosed angle brackets

**Pull request merged**